libs/api/domains/social-insurance/src/lib/resolvers/medicalDocuments.resolver.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+  type User,
+} from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { FeatureFlagGuard } from '@island.is/nest/feature-flags'
+import { UseGuards } from '@nestjs/common'
+import { Query, Resolver } from '@nestjs/graphql'
+import { RehabilitationPlanModel } from '../models/medicalDocuments/rehabilitationPlan.model'
+import { SocialInsuranceService } from '../socialInsurance.service'
+
+@Resolver()
+@UseGuards(IdsUserGuard, ScopesGuard, FeatureFlagGuard)
```

Comment:
Nitpick: You don't seem to be using any feature flag decorators so the guard is unnecessary